### PR TITLE
Add info about how Frontend communicates with clients and services

### PR DESCRIPTION
**Pull request by matthew-white**

### CODE REVIEW

**Review by ktuite — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -41,18 +41,20 @@ Finally, run NGINX by changing the working directory to the root directory of th
...
+ODK Central Frontend communicates with ODK Central Backend in part using a session cookie. The cookie is `Secure` and `HttpOnly`, but will be sent over HTTP on localhost. ODK Central Frontend also interacts directly or indirectly with data collection clients and with services:
+
+- To upload an XLSForm, run [pyxform-http](https://github.com/getodk/pyxform-http). ODK Central Frontend communicates with pyxform-http through ODK Central Backend.
+- You can use ODK Collect to scan an app user QR code and collect data. One option to do so is to use [`ngrok`](https://ngrok.com/download). ODK Central Frontend is available on port 8989, so you can run `ngrok http 8989` to expose a temporary HTTPS URL that you can use. You will also need to set the `default.env.domain` property in [`config/default.json`](https://github.com/getodk/central-backend/blob/master/config/default.json) to the HTTPS URL, then restart ODK Central Backend if it is already running.
```
Comment (lognaturel) [inline README.md:51]: Does this work for making submissions? I vaguely remember it didn't.

File: README.md [hunk 2]
```diff
@@ -41,18 +41,20 @@ Finally, run NGINX by changing the working directory to the root directory of th
...
+- The development `nginx.conf` is not configured to work with Enketo.
```
Comment (lognaturel) [inline README.md:52]: You provide some context on what the other things you mention are. You might want to say something like "Enketo is a web form engine used to show form previews and allow for web-based data entry. The development `nginx.conf` is not configured to work with Enketo."

File: README.md (hunk 1 above)
Comment (matthew-white) [reply README.md]: I just double-checked submitting from Collect and was able to do so! 📩

**Comment by matthew-white** [comment]
@lognaturel, this should be ready for another look.

File: README.md (hunk 1 above)
Comment (lognaturel) [reply README.md]: 🙏 That makes life a lot easier, then!

**Review by lognaturel — APPROVED** [review]

**Pull request merged**